enterprise/server/cmd/ci_runner/main.go

```diff
@@ -489,127 +492,78 @@ func run() error {
 		return nil
 	}
 
-	var buildEventReporter *buildEventReporter
-	if *reportLiveRepoSetupProgress {
-		ber, err := newBuildEventReporter(ctx, *besBackend, ws.buildbuddyAPIKey, *invocationID, *workflowID != "" /*=isWorkflow*/)
-		if err != nil {
-			return err
-		}
-		buildEventReporter = ber
-		if err := buildEventReporter.Start(ws.startTime); err != nil {
-			return status.WrapError(err, "could not publish started event")
-		}
+	if err := buildEventReporter.Start(ws.startTime); err != nil {
```

Comment:
unrelated to this PR, but I wonder if we should publish the `Started` event before we publish any progress events during repo setup? Wondering if this could lead to the same issue as https://github.com/buildbuddy-io/buildbuddy-internal/issues/1259

Reply:
After this PR, the `Started` event will always be sent as the first event. `buildEventReporter.Start` publishes the `Started` event. The repo setup and everything else happens after this.

Reply:
ah, I thought that was the case at first, but then somehow confused myself. Nice :+1: